### PR TITLE
[AND-2394] bugfix:Add MoPub click track for banner.

### PR DESCRIPTION
**Pull request by HuFangshuai**
This PR have been created for bug fix: https://vungle.atlassian.net/browse/AND-2394

### CODE REVIEW

File: Vungle/src/main/java/com/mopub/mobileads/VungleBanner.java [hunk 1]
```diff
@@ -63,6 +63,8 @@ protected void loadBanner(Context context, CustomEventBannerListener customEvent
         mCustomEventBannerListener = customEventBannerListener;
         pendingRequestBanner.set(true);
 
+        setAutomaticImpressionAndClickTracking(false);
```
Comment (palrahul) [inline Vungle/src/main/java/com/mopub/mobileads/VungleBanner.java:66]: should this also be set for interstitial , I know that this came as suggestion from MoPUb for Banner, but might be relevant for Interstitial as well

File: Vungle/src/main/java/com/mopub/mobileads/VungleBanner.java [hunk 2]
```diff
@@ -206,6 +208,10 @@ public void onAdEnd(@NonNull String placementReferenceId, boolean wasSuccessfulV
                 mIsPlaying = false;
                 sVungleRouter.removeRouterListener(mPlacementId);
                 mVungleRouterListener = null;
```
Comment (palrahul) [inline Vungle/src/main/java/com/mopub/mobileads/VungleBanner.java:210]: not related to the changes here but seems like we dont uniformly follow on what thread we are making these callbacks, some have `mHandler.post` and some do not, could this create issues with Pub getting different events posted back on different threads

**Review by palrahul — APPROVED** [review]
overall LGTM, some minor suggestions

Q: Have we tested and confirmed that when Banner is clicked, theres no discrepancy in that we hit this newly added click callback, as well as have it in our internal reportAd

File: Vungle/src/main/java/com/mopub/mobileads/VungleBanner.java (hunk 1 above)
Comment (HuFangshuai) [reply Vungle/src/main/java/com/mopub/mobileads/VungleBanner.java:66]: ok, fixed.

File: Vungle/src/main/java/com/mopub/mobileads/VungleBanner.java (hunk 2 above)
Comment (HuFangshuai) [reply Vungle/src/main/java/com/mopub/mobileads/VungleBanner.java:210]: ok, fixed it follow main UI thread for MoPub callbacks

**Comment by HuFangshuai** [comment]
> overall LGTM, some minor suggestions
> 
> Q: Have we tested and confirmed that when Banner is clicked, theres no discrepancy in that we hit this newly added click callback, as well as have it in our internal reportAd

I tested isCTAClicked following design in PlayAdCallback.onAdEnd() from SDK when I click MREC ad, and /report_ad seems have recorded the correct action:
clickedThrough": ["videoLength", "videoViewed", "videoViewed", "videoViewed", "videoViewed", "videoViewed", "videoViewed", "videoViewed", "videoViewed", "videoViewed", "videoViewed", "videoViewed", "videoViewed", "videoViewed", "videoViewed", "videoViewed", "download", "mraidOpen", "videoViewed", "videoViewed", "videoViewed", "videoViewed", "download", "mraidOpen", "videoViewed", "videoViewed", "videoViewed", "videoViewed"]

**Pull request merged**